### DISCUSSION TITLE
Help interpreting this modifier key symbol ⌨

### DISCUSSION THREAD

**Discussion by illettante** [mucommander/mucommander#598] [Q&A]
Hi commμnity,
Just wondering what key this modifier key symbol (⌨) is. It's used for _mark_ and _unmark_, and i'm not familiar with it (which just means i looked [here](https://support.apple.com/en-au/guide/mac-help/cpmh0011/mac) and couldn't find it. What's its longhand name?

**Comment by ahadas** [comment]
hi @illettante , I'm not sure where did you see that key symbol but if it's the one used for mark/unmark then it's either the space key or insert key

**Reply by illettante** [reply]
It shows up a couple of times, I think it's just mac's chosen symbol for whatever it is. Sorry, I wasn't exact enough in my first post, it's used for mark and unmark _group_
![image](https://user-images.githubusercontent.com/94096440/162138409-a539f3c7-a32c-4e8b-ad16-f485f5e558ba.png)

**Reply by illettante** [reply]
Ok so i played around with adding secondary shortcuts to make entries appear in the resulting action_keymap.xml. It looks like those shortcuts are actually unmodified, and based on the names of the keys ('ADD' and 'SUBTRACT' as opposed to 'MINUS' and 'shift EQUALS' i reckon the ⌨ symbol is Mac's way of indicating that it's the + and - keys on the numpad rather on the typewriter set. Can't test that theory bc I don't have a numpad. Won't mark this as solved yet though, in case anyone can confirm.
